Adyen/UI/List/ListHeaderView.swift

```diff
@@ -28,23 +28,84 @@ internal final class ListHeaderView: UIView {
         fatalError("init(coder:) has not been implemented")
     }
     
+    internal var headerItem: ListSectionHeader? {
+        didSet {
+            updateItem()
+        }
+    }
+    
+    private func updateItem() {
+        guard let item = headerItem else { return }
+        backgroundView?.backgroundColor = item.style.backgroundColor
+        contentView.backgroundColor = item.style.backgroundColor
+        titleLabel.adyen.apply(item.style.title)
+        titleLabel.accessibilityIdentifier = ViewIdentifierBuilder.build(scopeInstance: "Adyen.ListHeaderView.\(item.title)",
+                                                                         postfix: "titleLabel")
+        titleLabel.text = item.title.uppercased()
+        
+        trailingButton.adyen.apply(item.style.trailingButton)
+        
+        updateTrailingButtonTitle(with: item)
+    }
+    
+    internal var isEditing: Bool = false {
+        didSet {
+            updateTrailingButtonTitle(with: headerItem)
+        }
+    }
+    
+    private func updateTrailingButtonTitle(with item: ListSectionHeader?) {
+        guard let item = headerItem else { return }
+        let localizedEdit = Bundle.Adyen.localizedEditCopy
+        let localizedDone = Bundle.Adyen.localizedDoneCopy
+        let localizedTitle = isEditing ? localizedDone : localizedEdit
+        switch item.editingStyle {
```

Comment:
What if merchant wants to change this label or use custom translation?

Reply:
as discussed offline, we don't support forced locale at the moment and its not likely merchant would want to override those translations.